src/returns/date/season-in-abstraction-period.js

```diff
@@ -0,0 +1,81 @@
+'use-strict';
+
+const moment = require('moment');
+const summer = {
+  startDay: '01',
```

Comment:
Do these need to be strings or can they be integers?

Reply:
done